frameworks/kafka/docs/configure.md

```diff
@@ -294,6 +303,45 @@ To configure it:
 
 This configuration option cannot be changed after installation.
 
+## TLS
+
+It is possible to expose Kafka over a secure TLS connection.
+
+**DC/OS CLI options.json**:
+```
+{
+    "service": {
+        "tls": true
+    }
+}
+```
+
+Once the TLS is enabled, Kafka will do inter-node communication over secure TLS connections. Additional TLS port named `broker-tls` will be available for clients connecting to the service. Only the [`TLSv1.2`](https://www.ietf.org/rfc/rfc5246.txt) is supported. The non-TLS `broker` port will be still available.
+
+Enabling the TLS is possible only in `permissive` or `strict` cluster security modes. Both modes **require** a [service account](https://docs.mesosphere.com/service-docs/kafka/kafka-auth/). Additionally, the service account **must have** the `dcos:superuser` permission. If the permission is missing the Kafka scheduler will not abe able to provision TLS artifacts.
```

Comment:
It is possible to launch Kafka with TLS in both `permissive` and `strict` security modes.